lib/AST/ASTPrinter.cpp

```diff
@@ -4785,12 +4787,24 @@ void PrintAST::visitErrorExpr(ErrorExpr *expr) {
   Printer << "<error>";
 }
 
-void PrintAST::visitTernaryExpr(TernaryExpr *expr) {}
+void PrintAST::visitTernaryExpr(TernaryExpr *expr) {
+  visit(expr->getCondExpr());
+  Printer << " ? ";
+  visit(expr->getThenExpr());
+  Printer << " : ";
+  visit(expr->getElseExpr());
```

Comment:
As mentioned, you'll want to check `getCondExpr` and `getElseExpr` to make sure they're non-null before printing